.circleci/config.yml

```diff
@@ -49,26 +49,18 @@ jobs:
             # choose available python versions from pyenv
             pyenv_py_ver=""
             case << parameters.NRN_PYTHON_VERSION >> in
-              38) pyenv_py_ver="3.8.10" ;;
-              39) pyenv_py_ver="3.9.13" ;;
-              310) pyenv_py_ver="3.10.11" ;;
-              311) pyenv_py_ver="3.11.7" ;;
-              312) pyenv_py_ver="3.12.0" ;;
+              38) pyenv_py_ver="3.8" ;;
+              39) pyenv_py_ver="3.9" ;;
+              310) pyenv_py_ver="3.10" ;;
+              311) pyenv_py_ver="3.11" ;;
+              312) pyenv_py_ver="3.12" ;;
               *) echo "Error: pyenv python version not specified!" && exit 1;;
             esac
 
-            # install python dependencies: .10 is not available pyenv
-            if [ "<< parameters.NRN_PYTHON_VERSION >>" == "310" ]; then
-              sudo apt install software-properties-common -y
-              sudo add-apt-repository ppa:deadsnakes/ppa -y
-              sudo apt install python3.10 libpython3.10 python3.10-venv
-              export PYTHON_EXE=$(which python3.10)
-            else
-              cd /opt/circleci/.pyenv/plugins/python-build/../.. && git pull && cd -
-              env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install $pyenv_py_ver --force
-              pyenv global $pyenv_py_ver
-              export PYTHON_EXE=$(which python)
-            fi
+            cd /opt/circleci/.pyenv/plugins/python-build/../.. && git fetch --all && git checkout -B master origin/master && cd -
+            env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install $pyenv_py_ver --force
```

Comment:
@alkino : I am only wondering about being on the master. Do you think that won’t be too “unstable”?

Reply:
We were already on the master but no more because in the new version we were on a detached head.